fvm/script.go

```diff
@@ -198,6 +199,11 @@ func (executor *scriptExecutor) executeScript() error {
 	rt := executor.env.BorrowCadenceRuntime()
 	defer executor.env.ReturnCadenceRuntime(rt)
 
+	// TODO: how to clean up for the next script
```

Comment:
What do you mean by this?

are you worried  `executor.ctx.EVMEnabled` would change?

Reply:
Yeah, but maybe thats a unnecessary aspect to be worried about.

Reply:
I don't think its needed to expect a config change like this.

Reply:
right, remove the comment.